boa/build.py

```diff
@@ -468,6 +470,7 @@ def write_build_scripts(m, script, build_file):
 
     with open(work_file, 'w') as bf:
         # bf.write('set -ex\n')
+        bf.write('#!/usr/bin/env bash\n\n')
```

Comment:
then we could leave out this line

Reply:
This line might not be needed actually, I'll try next week